Egy Le Rendszere/Ajánlatok

```diff
@@ -0,0 +1,25 @@
+Nézvén a magaslatból
+Látod hogy a hajók messze
+Látod sasszemmel
+A hullámok folyton csapkodnak
+Ha te vagy a jelfény
+Tiéd a munkásosztály
+De hogyha válasz kéne
+Jobb lesz ha adsz egy darab segget!
+Adj egy darab segget
+
+Vigyázz, delej utáni tippek
```

Comment:
tippek -> ajánlat? (itt is van +1 szótag hely)

Reply:
az angol nyelv hülyesége miatt inkább sug-ges-tions mint sug-ges-ti-ons

Reply:
ok a tippek helyett az ötlet lett